spec/spec_helper.rb

```diff
@@ -14,25 +29,30 @@
   config.filter_sensitive_data('API_TOKEN') { ENV['MASTER_JWT'] }
 end
 
-mode = ENV['MODE'] || 'unit'
-
 $LOAD_PATH.unshift File.expand_path('..', __FILE__)
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 
-require 'rspec'
-require 'rack/test'
-require 'faker'
-require 'auth0'
-
 Dir['./lib/**/*.rb'].each { |f| require f }
 Dir['./spec/support/**/*.rb'].each { |f| require f }
 Dir['./spec/support/*.rb'].each { |f| require f }
 
+require 'rspec'
```

Comment:
cosmetic but, shouldn't this be on the top of the file instead of flying here?

Reply:
Actually no ... `config.include Credentials` needs to happen after the file requires